vaadin-popover-flow-parent/vaadin-popover-flow/src/test/java/com/vaadin/flow/component/popover/PopoverTest.java

```diff
@@ -348,4 +356,23 @@ private void testStaticSettersThreadsSafety(Runnable tester) {
         executorService.shutdown();
         Mockito.verify(current, times(1)).addUIInitListener(Mockito.any());
     }
+
+    @Test
+    public void setTarget_openModal_popoverIsAttachedToUi() {
```

Comment:
Wouldn't make more sense to add this test to the [`PopoverAutoAddTest`](https://github.com/vaadin/flow-components/blob/main/vaadin-popover-flow-parent/vaadin-popover-flow/src/test/java/com/vaadin/flow/component/popover/PopoverAutoAddTest.java) class?

Reply:
Thanks, done